sky/serve/controller.py

```diff
@@ -152,7 +152,7 @@ def configure_logger():
         logger.info('SkyServe Controller started on '
                     f'http://localhost:{self._port}')
 
-        uvicorn.run(self._app, host='localhost', port=self._port)
+        uvicorn.run(self._app, host='0.0.0.0', port=self._port)
```

Comment:
It would be nice to add a comment here for why `0.0.0.0` instead of `localhost` is used.